### PR TITLE
feat(DeviceSettings): info alert with a link to the documentation

### PR DESCRIPTION
**Pull request by MacDada**
Screenshot 2024-05-14 at 10 26 29

The link to the docs allows the user to quickly find what are the settings options for.

I personally was looking for what is "Disabled" option actually doing.

I wasn't sure how to style it -> if there is a better (maybe some kind of "standard") way of injecting this kind of info alerts, I will happily improve the PR :-)

### CODE REVIEW

File: src/components/device-page/DeviceSettings.tsx [hunk 1]
```diff
@@ -54,14 +54,40 @@ export class DeviceSettings extends Component<DeviceSettingsProps, DeviceSetting
...
         return (
-            <Form
-                schema={schema}
-                formData={data}
-                onSubmit={this.updateConfig}
-                onChange={this.onFormChange}
-                uiSchema={uiSchema}
-                fields={{ TitleField, DescriptionField }}
-            />
+            <div>
```
Comment (MacDada) [inline src/components/device-page/DeviceSettings.tsx]: I will humbly listen to suggestions on how to refactor this to fix https://codebeat.co/projects/github-com-nurikk-zigbee2mqtt-frontend-dev/pull_requests/734366

Anything similar in the source code, that I can look at for inspiration? I'm pretty new to React ;)

Comment (S33G) [reply src/components/device-page/DeviceSettings.tsx]: There's already libraries to handle icons
e.g.

```
<i class="fa-solid fa-circle-info"></i>
```
https://fontawesome.com/icons/circle-info?f=classic&s=solid

Comment (S33G) [reply src/components/device-page/DeviceSettings.tsx]: https://www.npmjs.com/package/@fortawesome/fontawesome-free

Comment (MacDada) [reply src/components/device-page/DeviceSettings.tsx]: https://github.com/nurikk/zigbee2mqtt-frontend/pull/2020/commits/2e2272f9ac9d7894f2311e8ee6867af510b8caa5

Comment (MacDada) [reply src/components/device-page/DeviceSettings.tsx]: Unfortunately, the function is still "too long": https://codebeat.co/projects/github-com-nurikk-zigbee2mqtt-frontend-dev/pull_requests/734366

Comment (MacDada) [reply src/components/device-page/DeviceSettings.tsx]: I could just get rid of some whitespace, but I'd rather go into a more proper direction, like extracting something like `<InfoAlert message="blahblah" />`

Comment (MacDada) [reply src/components/device-page/DeviceSettings.tsx]: https://github.com/nurikk/zigbee2mqtt-frontend/pull/2020/commits/06b2fdae70352b05fda115a3627abc685d92ce0c

File: src/components/ReadTheDocsInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+export const ReadTheDocsInfo = (props: { docsUrl: string }): JSX.Element => {
+    const { docsUrl } = props;
+    return (
+        <div className="card alert alert-info" role="alert">
+            <div className="card-body">
+                <i className="fa-solid fa-circle-info fa-2xl me-2"></i>
+
+                <a href={docsUrl} target="_blank" rel="noreferrer" className="alert-link align-middle">
+                    Read about this in the documentation…
```
Comment (nurikk-sa) [inline src/components/ReadTheDocsInfo.tsx]: this calls for translation

Comment (MacDada) [reply src/components/ReadTheDocsInfo.tsx]: https://github.com/nurikk/zigbee2mqtt-frontend/pull/2020/commits/0eb757b162caf99b85d638a178f9af53852c9670

**Comment by nurikk** [comment]
Terrific job! Thanks

**Pull request merged**